src/pytest_servers/s3.py

```diff
@@ -1,10 +1,14 @@
+import sys
+
 import pytest
 
+LOCALHOST = "127.0.0.1" if sys.platform == "win32" else "localhost"
+
 
 class MockedS3Server:
     def __init__(
         self,
-        ip_address: str = "localhost",
+        ip_address: str = LOCALHOST,
```

Comment:
Let's just hardcode it to "127.0.0.1" here. That's what `ThreadedMotoServer` does.

Reply:
Yeah, I guess that's simpler and more explicit.